requirements.txt

```diff
@@ -50,8 +50,8 @@ Python-jose==3.3.0
 gitpython==3.1.27
 debugpy==1.6.0
 python-gitlab==3.3.0
-google-api-python-client==2.42.0
-google-auth==2.6.2
+google-api-python-client==2.43.0
```

Comment:
I have no idea why this is marked as a change, it is the same version as in `dev`.